jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/migration/MigrationManager.java

```diff
@@ -601,7 +601,10 @@ protected TimerManager getTimerManager(KieSession ksession) {
                                 List<TimerInstance> collected = new ArrayList<>();
                                 for (Long timerId : timers) {
                                     TimerInstance timerInstance = timerManager.getTimerMap().get(timerId);
-
+                                    if(timerInstance==null){
```

Comment:
Format: add blanks around the parenthesis:
```suggestion
                                    if (timerInstance==null) {
```